gsa/src/web/components/icon/downloadkeyicon.js

```diff
@@ -0,0 +1,36 @@
+/* Copyright (C) 2019 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+
+import SvgIcon from './svgicon';
+
+import {ReactComponent as Icon} from './svg/dl_key.svg';
+
+const DownloadKeyIcon = props => (
+  <SvgIcon
+    {...props}
```

Comment:
The other icons have titles. Is this on purpose that the DownloadKeyIcon doesn't? Is it passed via props?